tools/ptm_lanlgeomag.c

```diff
@@ -134,12 +142,18 @@ int main( int argc, char *argv[] ){
     strcpy( arguments.ExtModel, "T96" );
     arguments.Time = 120000;
     arguments.Verbose = 1;
+    arguments.DeltaT = 300;
+    arguments.TTot = 12;
 
     /*
      *  Parse CmdLine arguments and options
      */
     argp_parse (&argp, argc, argv, 0, 0, &arguments);
+    strcpy(dirname, arguments.args[0]);
     verbose = arguments.Verbose;
+    int t_tot = arguments.TTot + 1; // +1 because a single timestep needs bounding fields
```

Comment:
This will trigger all the calculations twice, which is unnecessary/slow... in `ptm_preprocessing.convertSWMF` this is handled by simply saving the time step twice (but not processing anything twice).

I'd suggest handling this similarly to `ptm_preprocessing`.

Reply:
That makes sense, I honestly had never just run one timestep before and noticed that my "do it all" shell script would fail if you wanted just one timestep. I should have looked into why that was and where it failed, but I was just trying to get something done quick and dirty. I'll make the change so that it handles it the same way as ptm_preprocessing.